versioned_docs/version-2020fa/lecture6.md

```diff
@@ -3,4 +3,473 @@ id: lecture6
 title: Lecture 6
 ---
 
-Coming soon!
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+[Lecture Slides](https://docs.google.com/presentation/d/1ID2ElICd0D4uh7KBKWvIgf7s60PLTI1X4LPg2Saawvw/edit?usp=sharing)
+
+[Lecture Video](https://drive.google.com/file/d/1r9g1X9XGGqTRfNX7gkg0bprLB7efPbNp/view?usp=sharing)
+
+[Assignment 5](/docs/assignment5) due **11/04 3:59pm**
+
+## Conditional Rendering
+
+Sometimes we only want things to render when a certain condition is met. For
+example, only display text when we meet a certain condition. React has
+conditional rendering to make this very simple.
+
+<Tabs
+groupId="lang"
+defaultValue="ts"
+values={[
+{ label: 'TypeScript', value: 'ts', },
+{ label: 'JavaScript', value: 'js', },
+]
+}>
+<TabItem value="ts">
+
+```tsx title="PrelimTime.tsx"
+import React from 'react';
+
+export default ({ prelimToday }: { readonly prelimToday: boolean }) => {
```

Comment:
We should avoid anonymous default export here. See https://github.com/cornell-dti/trends-in-web-dev-website/pull/169. Same for everything below.